plugins/prow-job/skills/prow-job-list-qe-jobs/generate_report.py

```
@@ -0,0 +1,207 @@
+from collections import Counter
+
+
+def load_template(template_file):
+    """Load HTML template from file."""
+    try:
+        with open(template_file, 'r') as f:
+            return f.read()
+    except FileNotFoundError:
+        print(f"Error: Template file '{template_file}' not found.")
+        sys.exit(1)
+    except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)